build.sbt

```diff
@@ -811,56 +811,66 @@ lazy val standaloneApi = (project in engine("standalone/api")).
   settings(commonSettings).
   settings(
     name := "nussknacker-standalone-api"
-  ).dependsOn(baseEngineApi)
+  ).dependsOn(liteEngineApi)
 
-lazy val baseEngineApi = (project in engine("base/api")).
+lazy val liteEngineApi = (project in engine("base/api")).
   settings(commonSettings).
   settings(
-    name := "nussknacker-baseengine-api",
+    name := "nussknacker-lite-api",
   ).dependsOn(api)
 
 lazy val liteBaseComponents = (project in engine("base/components/base")).
   settings(commonSettings).
   settings(assemblySampleSettings("liteBaseComponents.jar"): _*).
   settings(
     name := "nussknacker-lite-base-components",
-  ).dependsOn(baseEngineApi % "provided")
+  ).dependsOn(liteEngineApi % "provided")
 
 
-lazy val baseEngineRuntime = (project in engine("base/runtime")).
+lazy val liteEngineRuntime = (project in engine("base/runtime")).
   settings(commonSettings).
   settings(
-    name := "nussknacker-baseengine-runtime",
+    name := "nussknacker-lite-runtime",
     libraryDependencies ++= {
       Seq(
         "io.dropwizard.metrics5" % "metrics-core" % dropWizardV,
         "io.dropwizard.metrics5" % "metrics-influxdb" % dropWizardV,
         "com.softwaremill.sttp.client" %% "core" % sttpV,
       )
     },
-  ).dependsOn(baseEngineApi, interpreter, testUtil % "test")
+  ).dependsOn(liteEngineApi, interpreter, testUtil % "test")
 
-lazy val nuKafkaEngineBinTest: Project = (project in engine("base/nu-kafka-engine-bin-test")).
+lazy val liteKafkaEngineBinTest: Project = (project in engine("base/kafka-runtime-bin-test")).
   configs(IntegrationTest).
   settings(itSettings()).
   enablePlugins().
   settings(commonSettings).
   settings(
-    name := "nussknacker-baseengine-runtime-kafka",
+    name := "nussknacker-lite-runtime-kafka-bin-test",
     IntegrationTest / Keys.test := (IntegrationTest / Keys.test).dependsOn(
-      kafkaBaseEngineRuntime / Universal / stage
+      liteKafkaEngineRuntime / Universal / stage
     ).value,
     libraryDependencies ++= Seq(
       "commons-io" % "commons-io" % commonsIOV
     )
   ).dependsOn(interpreter % "it", kafkaUtil % "it", testUtil % "it", kafkaTestUtil % "it")
 
-lazy val kafkaBaseEngineRuntime: Project = (project in engine("base/kafka")).
+// TODO: move to base/kafka/api
+lazy val liteKafkaEngineApi = (project in engine("base/kafka-api")).
+  settings(commonSettings).
+  settings(
+    name := "nussknacker-lite-kafka-api",
+    libraryDependencies ++= Seq(
+      "org.apache.kafka" % "kafka-clients" % kafkaV
+    )
+  ).dependsOn(api)
+
+lazy val liteKafkaEngineRuntime: Project = (project in engine("base/kafka")).
   settings(commonSettings).
   // TODO: provide docker distribution
   enablePlugins(SbtNativePackager, JavaServerAppPackaging).
   settings(
-    name := "nussknacker-baseengine-runtime-kafka",
+    name := "nussknacker-lite-runtime-kafka",
```

Comment:
nussknacker-lite-kafka-api vs 
nussknacker-lite-runtime-kafka
I think we're a bit inconsistent here? nussknacker-lite-kafka-runtime?

Reply:
good point, will change